modules/log-collector-rsyslog-server.adoc

```diff
@@ -6,7 +6,11 @@
 [id="log-collector-rsyslog-server_{context}"]
```

Comment:
Note the file name and ref that it is just a 'syslog' server, not an 'rsyslog'.  Rsyslog is actually a specific product that is designed around the syslog protocol

Reply:
Renamed to syslog